src/main/java/com/internhub/data/positions/scrapers/candidates/QueuedCandidate.java

```diff
@@ -0,0 +1,13 @@
+package com.internhub.data.positions.scrapers.candidates;
+
+public class QueuedCandidate {
```

Comment:
I honestly don't see the reason for this abstraction layer either as all it does is wrap around a URL. Why not just a URL in the queue instead of a QueuedCandidate, what is the benefit for this layer?